sunpy/map/tests/test_mapbase.py

```diff
@@ -262,6 +267,33 @@ def test_rotation_matrix_cd_cdelt():
     np.testing.assert_allclose(cd_map.rotation_matrix, np.array([[0., -1.], [1., 0]]))
 
 
+def test_rotation_matrix_cd_cdelt():
```

Comment:
This new test has the same name as an already existing one and I think it could be greatly simplified. 

```python
def test_norm_cmap_kwargs(generic_map):
    cmap = 'cool'
    from matplotlib import colors, pyplot as plt
    norm = colors.Normalize(vmin=1., vmax=1.)
    smap = sunpy.map.Map(generic_map.data, generic_map.meta, cmap=cmap, norm=norm)
    assert smap.cmap == plt.get_cmap(cmap)
    assert smap.norm == norm 
```

We probably need one additional test for the new deprecation warning as well 😁